packages/itwin/measure-tools/package.json

```diff
@@ -44,42 +44,40 @@
     "url": "http://www.bentley.com"
   },
   "peerDependencies": {
-    "@itwin/appui-abstract": "^4.7.0",
-    "@itwin/appui-layout-react": "^4.4.0",
-    "@itwin/appui-react": "^4.4.0",
-    "@itwin/components-react": "^4.4.0",
-    "@itwin/core-bentley": "^4.7.0",
-    "@itwin/core-common": "^4.7.0",
-    "@itwin/core-frontend": "^4.7.0",
-    "@itwin/core-geometry": "^4.7.0",
-    "@itwin/core-react": "^4.4.0",
+    "@itwin/appui-abstract": "^4.10.0",
+    "@itwin/appui-react": "^4.17.0",
+    "@itwin/components-react": "^4.17.0",
+    "@itwin/core-bentley": "^4.10.0",
```

Comment:
@aruniverse  Since these are peerDependencies, shouldnt that cause this to be a major change ? Basically it means that if we want to use this new version, we need to bump our dependencies to consume it and make sure that we are to the latest dependencies, which if no code change is required, means that there is no real reasons to change the iTwin.js dependencies. (on top of that, I think it shouldnt be hidden under the iTwinUI V3 changes)

It would at least make this 1.0.0, and make this package more easily manageable with versions, rather than 0.x version which is ignored by package management.

Reply:
Per my other comment, if there are no new APIs being used, the peer deps should probably just remain the same.

Reply:
Thats very fair, while these are "breaking" since this pkg is 0.x its fair imo.
I do not want to bump this to 1.0 until we have clear idea of ownership of this pkg, also with both appui and core majors around the corner we'd have to do a 2.0 and 3.0 release shortly back to back. 

I reverted bumping the core peer deps. The AppUI you need to bump to a minimum of 4.10 to pull in iTwinUI@3. And at that point if youre bumping it, might as well go to the latest. This also helps us verify we're not using any deprecated APIs that will be removed in AppUI@5.

DevDeps have been updated to latest minors and that should be fine